src/xdocs/config_imports.xml

```diff
@@ -603,11 +603,26 @@ import android.*;
           </tr>
           <tr>
             <td>illegalPkgs</td>
-            <td>packages to reject</td>
+            <td>Packages to reject, by default checks if import is the part of package. If <b>regexp</b> variable is set, then list of packages will be
+                interpreted as regular expressions. Note, all properties for match will be used. Order of matching: <b>illegalPkgs</b>, <b>illegalClasses</b></td>
```

Comment:
Why is order important? We are looking for forbidden packages/classes. Order doesn't matter if one is found somewhere.
Please remove.